lambdas/user-list-search-indexing/src/helper.ts

```diff
@@ -0,0 +1,86 @@
+import { UserItemsSqsMessage, UserListImportSqsMessage } from './types';
+import { nanoid } from 'nanoid';
+import fetch from 'node-fetch';
+import { config } from './config';
+
+export const processUserImport = async (
```

Comment:
Mind slapping some short docstrings on these guys to help future readers?

Reply:
done!